blogs/models.py

```diff
@@ -0,0 +1,22 @@
+from django.contrib.auth.models import User
+from django.db import models
+
+
+class Blogs(models.Model):
+    title = models.CharField(max_length=100, null=False)
+    content = models.CharField(max_length=400, null=False)
+    author = models.ForeignKey(User, to_field='username', on_delete=models.CASCADE)
```

Comment:
Due to this `on_delete=models.CASCADE` all blogs of a user will be deleted when that user is deleted from the system. If this is your use case then it is perfect otherwise you might want to use something else according to your use case. You can look for docs for a detailed explanation of all the options.

https://docs.djangoproject.com/en/3.2/ref/models/fields/#django.db.models.ForeignKey.on_delete